ciri/consensus/snapshot/local_snapshots/local_snapshots_manager.c

```diff
@@ -152,6 +164,10 @@ retcode_t iota_local_snapshots_manager_stop(local_snapshots_manager_t *const lsm
     return RC_OK;
   }
 
+  if (lsm->conf->local_snapshots.pruning_is_enabled) {
+    ERR_BIND_RETURN(iota_local_snapshots_pruning_service_stop(&lsm->ps), ret);
```

Comment:
We should not return in case of failure here. It would mean we never get to signal and join and cIRI would be stuck. We should just assign `ret` and return it at the end.

Reply:
it won't actually because there' `cond_handle_timedwait` blocking, 
but still, good catch